app-editors/atom/atom-1.7.3.ebuild

```diff
@@ -2,84 +2,361 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=6
+EAPI=5
```

Comment:
I didn't base my ebuild on the current tree one. No problem bumping to 6, though.